File: demo/src/main/java/com/example/Fecha.java
package com.example;

public class Fecha {
    private int dd, mm, aa;

    public Fecha() {}
    public Fecha(int dd, int mm, int aa) {
        this.dd = dd;
        this.mm = mm;
        this.aa = aa;
    }

    public void setDia(int dd) { this.dd = dd; }
    public void setMes(int mm) { this.mm = mm; }
    public void setA(int aa) { this.aa = aa; }

    public int getDia() { return dd; }
    public int getMes() { return mm; }
    public int getA() { return aa; }

    public String toString() {
        return dd + " - " + mm + " - " + aa;
    }
}

File: python/Fecha.py
class Fecha:
    def __init__(self, dd=0, mm=0, aa=0):
        self.dd = dd
        self.mm = mm
        self.aa = aa

    def set_dia(self, dd): self.dd = dd
    def set_mes(self, mm): self.mm = mm
    def set_ano(self, aa): self.aa = aa

    def get_dia(self): return self.dd
    def get_mes(self): return self.mm
    def get_ano(self): return self.aa

    def __str__(self):
        return f"{self.dd} - {self.mm} - {self.aa}"
